src/main/java/com/codecool/virtualpub/data/Drink.java

```diff
@@ -0,0 +1,34 @@
+package com.codecool.virtualpub.data;
+
+public class Drink {
+    private final String brand;
+    private final DrinkType drinkType;
+    private final int price;
+    private int amount;
```

Comment:
How will this amount get a value? If it starts based on drinkType then it could work.

Reply:
I forgot to add the amount initialization. Yes, it is going to get the value from drinkType amountOfPours field.